src/main/java/com/netflix/control/controllers/PIDController.java

```diff
@@ -62,44 +66,65 @@
    * Oscillation: High gain can exacerbate and even cause oscillation.
    * Chaos Kong: Increasing gain to accelerate scale ups.
    */
-  public PIDController(Double kp, Double ki, Double kd, Double deltaT, AtomicDouble dampener) {
+  public PIDController(Double kp, Double ki, Double kd, Double deltaT, AtomicDouble dampener, double integralDecay) {
     this.kp = kp;
     this.ki = ki;
     this.kd = kd;
     this.deltaT = deltaT;
     this.dampener = dampener;
+    this.integralDecay = integralDecay;
```

Comment:
should we make the enabling of decay configurable? that way we can a/b test its impact

Reply:
Do you mean set `DEFAULT_INTEGRAL_DECAY` to 1.0? That was be exactly the same as it was before.

Reply:
Right leave the old constructor that defaults to 1.0 and if we want the decay we pass it in?

Reply:
Moved it so it's controlled by ClutchConfiguration. ClutchConfiguration will default it to 1.0. And it can be set by users.